### PR TITLE
feat: create more initial users with GWM and cleaning

### PR DESCRIPTION
**Pull request by obdulia-losantos**
Depends on https://github.com/eHealthAfrica/gateway-manager/pull/57 :heavy_check_mark: 

- Reorganized GWM solutions and services, updated `initialize.sh` (some of the command lines are now in a solution)
- Added more env var to GWM in order to create more users while creating a realm, updated `initialize.sh` with those en vars and added the lines for that.

@femilofin I'm not sure how to proceed with the new env vars because some of them will require some work from your side (secrets, secrets, secrets)

- `KEYCLOAK_INITIAL_USER_USERNAME` renamed to `INITIAL_USER_USERNAME`, defaults to `user`.
- `KEYCLOAK_INITIAL_USER_PASSWORD`renamed to `INITIAL_USER_PASSWORD`, defaults to `password`.
- `INITIAL_SUPERUSER_USERNAME` *new*, defaults to `sys-admin`.
- `INITIAL_SUPERUSER_PASSWORD` *new*, defaults to `password`.
- `INITIAL_ADMIN_USERNAME` *new*, defaults to `admin`.
- `INITIAL_ADMIN_PASSWORD` *new*, defaults to `password`.

### CODE REVIEW

File: src/gateway-manager/templates/initialize.yaml [hunk 1]
```diff
@@ -6,27 +6,44 @@ data:
...
+
+    # create simple user
+    $gwm add_user $REALM $INITIAL_USER_USERNAME $INITIAL_USER_PASSWORD
+    $gwm add_user_group $REALM $INITIAL_USER_USERNAME "user"
 
     SOLUTIONS={{ .Values.gw.solutions }}
     for solution in "${SOLUTIONS[@]}"; do
+      if [[ "${RESET_REALM:-}" == "true" ]]; then
+        $gwm remove_solution $solution $REALM
+      fi
```
Comment (obdulia-losantos) [inline src/gateway-manager/templates/initialize.yaml:37]: @femilofin this does the trick to reset kong entries :wink:

Comment (femilofin) [reply src/gateway-manager/templates/initialize.yaml:37]: 👍

**Comment by obdulia-losantos** [comment]
@femilofin is it ready from your side?

**Comment by femilofin** [comment]
@obdulia-losantos Yeah, I approved already

**Pull request merged**